src/components/Card/components/CardInput/CardInput.tsx

```diff
@@ -189,12 +199,23 @@ class CardInput extends Component<CardInputProps, CardInputState> {
         this.kcpAuthenticationRef = ref;
     };
 
-    render({ loadingContext, hasHolderName, hasCVC, installmentOptions, enableStoreDetails }, { status, hideCVCForBrand, focusedElement }) {
+    render(
+        { loadingContext, hasHolderName, hasCVC, countryCode, installmentOptions, enableStoreDetails },
+        { status, hideCVCForBrand, focusedElement, issuingCountryCode }
+    ) {
         const hasInstallments = !!Object.keys(installmentOptions).length;
 
         let isOneClick: boolean = this.props.storedPaymentMethodId ? true : false;
         if (this.props.oneClick === true) isOneClick = true; // In the Drop-in the oneClick status may already have been decided, so give that priority
 
+        // If the merchant defined countryCode is 'KR'
+        let isKorea = countryCode?.toLowerCase() === 'kr';
+
+        // Override the value if issuingCountryCode is set
+        if (issuingCountryCode !== null) {
+            isKorea = issuingCountryCode.toLowerCase() === 'kr';
+        }
```

Comment:
I'd enforce the format of the `issuingCountryCode` and the `countryCode` in the `formatProps` in the Card element and then just check the value in one line.
```suggestion
        const isKorea = issuingCountryCode === 'KR' || countryCode === 'KR';
```

Reply:
That doesn't cover all scenarios and the tests fails

Reply:
No worries, we can always merge and review it later.